src/backend/gog/library.ts

```diff
@@ -138,16 +138,16 @@ export class GOGLibrary {
       return
     }
     this.refreshInstalled()
+    for (const game of libraryStore.get('games', []) as GameInfo[]) {
+      const copyObject = { ...game }
+      if (this.installedGames.has(game.app_name)) {
+        copyObject.install = this.installedGames.get(game.app_name)!
+        copyObject.is_installed = true
+      }
+      this.library.set(game.app_name, copyObject)
+    }
```

Comment:
@imLinguin is this useful or is there a better fix?

Reply:
This looks like could be written as a filter, no?
The loop with the filter might be slower but I think we would not need to copy the object, modify it and then put it in the library. 🤔

Reply:
Whole idea of copying is to avoid writing changes into `gog_store/library.json` I wanted it to only store game metadata while `installed.json` contains install informations